stock_picking_portal/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# Copyright (C) 2024 Cetmix OÜ
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Stock Picking Portal",
+    "summary": "Show customer delivery orders in portal",
+    "version": "16.0.1.0.0",
+    "depends": ["sale_stock"],
+    "author": "Cetmix, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "website": "https://github.com/OCA/stock-logistics-workflow",
+    "data": [
+        "security/ir.model.access.csv",
+        "views/res_config_settings.xml",
```

Comment:
Please check this [document](https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#file-naming) and rename files.

Reply:
Thanks,I'll fix it)